src/styles/card.scss

```
@@ -1,19 +1,29 @@
 .card {
+  padding: 15px 14px 14px;
   width: $item-width;
   background: #fff;
   border: 1px solid #f3f3f3;
   border-radius: 5px;
-  transition: box-shadow 0.5s;
+  transition: box-shadow 1s;
 
   &:hover {
     box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
   }
 
   &__img {
-    width: 160px;
-    height: 134px;
-    margin: 0 5px ;
-    margin-bottom: 42px;
+    width: 170px;
+    height: 164px;
+    padding: 12px 2px;
+    border: 3px solid #fff;
+    margin-bottom: 27px;
+    transition-property: transform;
+    transition-duration: 1s;
+    transition-timing-function: ease;
```

```suggestion
    transition: transform 1s ease;
```
Думаю, що краще записати це в одну властивість